old-ui/app/components/token-cell.js

```diff
@@ -81,9 +81,14 @@ function navigateTo (url) {
   global.platform.openWindow({ url })
 }
 
-function etherscanLinkFor (tokenAddress, address, network) {
+function explorerLinkFor (tokenAddress, address, network) {
   const prefix = prefixForNetwork(network)
-  return `https://${prefix}etherscan.io/token/${tokenAddress}?a=${address}`
+  const POAnetwokIDs = [77, 99]
+  if (POAnetwokIDs.includes(parseInt(network))) {
```

Comment:
Given that this is done in a lot of places in the code, maybe it's time to move it to some util function?